packages/lodestar-cli/src/cmds/validator/options/validatorDir.ts

```diff
@@ -0,0 +1,25 @@
+import path from "path";
+import {Options} from "yargs";
+
+export interface IValidatorDirArgs extends IGlobalArgs {
+  dbDir: string;
+  validatorDir: string;
+}
+
+import {IGlobalArgs} from "../../../options";
+
+export const dbDir = (args: IGlobalArgs): Options => ({
```

Comment:
I think dbDir should go in `validatorFile.ts` so it can be placed in the validatorDir and its default can be `path.join(args.validatorDir, "validator-db")`

Reply:
All options merged in the same file